src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -42,8 +45,9 @@
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 
-public class WorkspaceServiceTest extends BaseUnitTest {
-
+public class WorkspaceServiceTest extends BaseConnectedTest {
```

Comment:
Do we want this to be a connected test when most dependencies are mocked out? Even if we move to calling services directly instead of via HTTP the existing tests here are really only testing WM and its database.

Reply:
The new tests are using HTTP, so they do need to be connected tests. I think it's fine for connected tests to have a mix of real and mocked/stubbed dependencies. I think it might be more confusing to have multiple `WorkspaceServiceTest.java` files each with partial method coverage, but that is an option. Soon, workspace deletion will want to delete projects as well, which is another connected use case.